pkgs/applications/office/jabref/default.nix

```
@@ -5,7 +5,7 @@
 , makeDesktopItem
 , copyDesktopItems
 , unzip
-, xdg_utils
+, xdg_utils ? null
```

```suggestion
, xdg-utils
```

`? null' shouldn't be used and xdg_utils is an alias to xdg-utils